hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/ContextTransactionProcessorTest.java

```diff
@@ -224,7 +223,7 @@ void stillChargesHapiFeesOnAbort() {
                 .willReturn(HEVM_CREATION);
         given(processor.processTransaction(
                         HEVM_CREATION, rootProxyWorldUpdater, feesOnlyUpdater, hederaEvmContext, tracer, CONFIGURATION))
-                .willThrow(new AbortException(INVALID_CONTRACT_ID, SENDER_ID));
+                .willThrow(new HandleException(INVALID_CONTRACT_ID));
```

Comment:
Test could (should?) be renamed to note that this is "still charges fees for aborted contract _create_" (distinguished from the case of _call_).

Reply:
I removed the distinction between contract call and other types of transactions.  What I really wanted to do was to see if fees had or had not been collected before the transaction was aborted.

Reply:
I thought there still was a distinction at `ContextTransactionProcessor` L148?

Plus, charging fees is dependent on that config variable `chargeGasOnPreEvmException` so shouldn't this be one of the property-overriding tests? (and same for `stillChargesHapiFeesOnHevmException` and `stillChargesHapiFeesOnExceptionThrown`?  (In order to ensure it is set to `true`, at least.)

Reply:
Yes it turned out that this distinction was important for the existing test.  Thinking about it further, I realized that we only really care about `ContractCall` transactions because these are the only ones that have gas only fees charged.  So I take back my earlier comment above 😛.